package.json

```diff
@@ -41,10 +41,10 @@
     "publish-stable": "lerna version --force-publish",
     "lint-staged": "lint-staged",
     "next-with-deps": "./scripts/next-with-deps.sh",
```

Comment:
```suggestion
    "next-with-deps": "cross-env NEXT_TELEMETRY_DISABLED=1 ./scripts/next-with-deps.sh",
```

Reply:
@ijjk That script seems to call `pnpm next` which should call the other script that already has it defined

https://github.com/vercel/next.js/blob/f38ddf9f2bec217b0634cb740c88e81455c02575/scripts/next-with-deps.sh#L39

Reply:
Ah nice that works